sdk/src/main/java/software/amazon/awssdk/iot/EnumSerializer.java

```diff
@@ -1,6 +1,8 @@
 /**
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0.
+ *
+ * This file is generated.
```

Comment:
Is it?

Reply:
It's not generated generated. But it's copied from the codegen repo, so I added this phrase to indicate that we shouldn't change it in this repo.
I considered removing `EnumSerializer.java` and `Timestamp.java` files from codegen altogether, but decided to keep them because it has nothing to do with upgrading smithy version.
I can change `This file is generated` to something like `This file is copied from a third-party source, and should not be changed.`.

Reply:
Feels like it should get moved out of the generator entirely, but I suppose that can wait.

Reply:
I gave some thoughts to your comments, and you're absolutely right: it makes sense to dissociate these files from codegen right away. Which I did.